react/components/Sidebar.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react'
+import ReactDOM from 'react-dom'
+import PropTypes from 'prop-types'
+import { IconCaretRight } from 'vtex.styleguide'
+import { injectIntl, intlShape } from 'react-intl'
+
+import OutsideClickHandler from 'react-outside-click-handler'
+
+import MiniCart from '../MiniCart'
+
+const whiteColor = '#FFFFFF'
+
+/* SideBar component */
+class Sidebar extends Component {
+  componentDidMount() {
+    document.body.classList.add('vtex-minicart-sidebar-open')
+  }
+
+  componentWillUnmount() {
+    document.body.classList.remove('vtex-minicart-sidebar-open')
+  }
+
+  render() {
+    const { onBackClick, intl } = this.props
+
+    if (typeof window !== 'undefined') {
```

Comment:
Is this really necessary, manipulating the window and the body?

Reply:
Isso é necessário para deixar a sidebar na altura 100% da tela.

Reply:
if you say so...